core/build.gradle

```diff
@@ -291,7 +291,7 @@ task sparkPackagePom(dependsOn: shadowJar) { doLast {
   file("${rootProject.buildDir}/distributions").mkdirs()
   pom {
     project {
-      groupId 'SnappyDataInc'
+      groupId 'TIBCOSoftware'
```

Comment:
Spark packages weren't published for 1.2.0-HF-1 and not sure if we can do that for upcoming 1.3.0.